internals-js/src/federationSpec.ts

```diff
@@ -62,6 +62,7 @@ export const shareableDirectiveSpec = createDirectiveSpecification({
 export const tagDirectiveSpec = createDirectiveSpecification({
   name:'tag',
   locations: tagLocations,
+  repeatable: true,
```

Comment:
It's probably ok for this case, but is there a situation where we'd want a directive to be repeatable on the subgraph spec, but not the supergraph?